spec/factories/user.rb

```diff
@@ -0,0 +1,10 @@
+FactoryGirl.define do
+  factory :user do
+    email "user@email.com"
+    password "password"
+    password_confirmation "password"
+    salt { "asdasdastr4325234324sdfds" }
+    crypted_password { Sorcery::CryptoProviders::BCrypt.encrypt("secret", 
```

Comment:
Avoid using `{...}` for multi-line blocks.<br>Block body expression is on the same line as the block start.<br>Trailing whitespace detected.